lib/Models/Catalog/Esri/ArcGisFeatureServerCatalogItem.ts

```diff
@@ -136,15 +136,19 @@ export default class ArcGisFeatureServerCatalogItem extends MinMaxLevelMixin(
 
   @computed get imageryProvider() {
     // Don't return an imagery provider if we haven't loaded metadata yet
-    if (!this.loadMetadataResult) return undefined;
+    if (!this.strata.has(ArcGisFeatureServerStratum.stratumName)) {
+      return undefined;
+    }
```

Comment:
`loadMetadatResult` returns a new `Result` object after each call to `loadMetadata` which causes this method to recompute unnecessarily